twurl: init at 0.9.6

brechtcs opened the pull request:
###### Motivation for this change

Twurl gives more low-level access to the Twitter API (and is more up-to-date with the latest features) than the `t` tool that's already available in nixpkgs right now.


###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions: Ubuntu on WSL
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

legendofmiracles commented on maintainers/maintainer-list.nix:
```diff
@@ -1512,6 +1512,12 @@
     githubId = 1111035;
     name = "Break Yang";
   };
+  brecht = {
+    email = "brecht.savelkoul@alumni.lse.ac.uk";
+    github = "brechtcs";
+    githubId = 6107054;
+    name = "Brecht Savelkoul";
+  };
```
Separate commit: `maintainers: add brecht`

brechtcs replied:
Rebased and separated the changes into two commits. It's fine to keep both of them together in one PR though, right?

legendofmiracles replied:
That's not an issue.

brechtcs committed:
maintainers: add brecht

brechtcs committed:
twurl: init at 0.9.6

Twurl gives more low-level access to the Twitter API than the
`t` tool that's already available in nixpkgs right now
